BackofficeBundle/Resources/views/layout.html.twig

```diff
@@ -84,6 +84,6 @@
             pageSetUp();
         });
     </script>
-
+    {{ tinymce_init() }}
```

Comment:
Why init tinymce in all pages, you can't init it just in the page with form which uses tinymce ?